### PR TITLE
make query by label with limit work when total elements greater than 80w

### PR DESCRIPTION
**Pull request by zhoney**
Change-Id: I952d867fa306df2686b444053ceeecce0d3aed8c
fixed: #81

### CODE REVIEW

File: hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/GraphIndexTransaction.java [hunk 1]
```diff
@@ -511,7 +511,7 @@ private boolean needIndexForLabel() {
         } finally {
             locks.unlock();
         }
-        return ids;
+        return limit(ids, query);
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/GraphIndexTransaction.java]: Query query(ConditionQuery query)

File: hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseTables.java [hunk 2]
```diff
@@ -195,6 +196,20 @@ public void delete(Session session, BackendEntry entry) {
         public SecondaryIndex(String store) {
             super(joinTableName(store, TABLE));
         }
+
+        protected void parseRowColumns(Result row, BackendEntry entry,
+                                       Query query) throws IOException {
+            CellScanner cellScanner = row.cellScanner();
+            long num = query.offset() + query.limit();
```
Comment (javeme) [inline hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseTables.java]: call super if query is NO_LIMIT

Comment (zhoney) [reply hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseTables.java]: done

File: hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseTables.java [hunk 3]
```diff
@@ -195,6 +196,24 @@ public void delete(Session session, BackendEntry entry) {
         public SecondaryIndex(String store) {
             super(joinTableName(store, TABLE));
         }
+
+        @Override
+        protected void parseRowColumns(Result row, BackendEntry entry,
+                                       Query query) throws IOException {
+            if (query.limit() == Query.NO_LIMIT) {
+                super.parseRowColumns(row, entry, query);
```
Comment (javeme) [inline hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseTables.java]: return here

**Pull request merged**